Support for check of multiple cn's in altnames

scornelissen85 opened the pull request:
I want to have an option to verify multiple cn's in the altnames. With this change it is possible to add multiple times the option -n which will check, for each -n option given, if it exists in altnames. Once a check fail it will exit with critical status.

matteocorti commented:
Thanks for the patch. It does not pass the tests (it seems that it adds a space when testing a sign CN). I'll try to check when I find some time ...

scornelissen85 commented:
I've fixed the whitespace and I fixed that the multiple CN's are not passed as servername in openssl.

As extra bonus I added tests for the multiple CN's.
